float32 a
float32 b
---
float32 path1
float32 path2
